packages/server/lib/util/ci_provider.coffee

```diff
@@ -308,16 +329,24 @@ commitParams = ->
   _get(_providerCommitParams)
 
 commitDefaults = (existingInfo) ->
+  debug("git commit existing info")
+  debug(existingInfo)
+
   commitParamsObj = commitParams() or {}
-  debug("commit params object")
+  debug("commit info from provider environment variables")
   debug(commitParamsObj)
 
   ## based on the existingInfo properties
   ## merge in the commitParams if null or undefined
   ## defaulting back to null if all fails
-  _.transform existingInfo, (memo, value, key) ->
+  combined = _.transform existingInfo, (memo, value, key) ->
     memo[key] = _.defaultTo(value ? commitParamsObj[key], null)
 
+  debug("combined git and environment variables from provider")
```

Comment:
in the test runner all things are a single line.. so this should be `debug("... %o", object)`